### DISCUSSION TITLE
Navigating the Block Editor Handbook

### DISCUSSION THREAD

**Discussion by mburridge** [WordPress/developer-blog-content#96] [Topic Ideas]
The [Block Editor Handbook](https://developer.wordpress.org/block-editor/) can be intimidating even for an experienced developer, and is especially so for those new to block development.

There's a wealth of information in the handbook, but key knowledge is distributed across several different sections and can be difficult to find.

Furthermore, there is no clear guide to where stuff is or how to find your way through the handbook to build up your knowledge whilst ensuring that you don't miss anything crucial.

I propose an article that provides a roadmap or path through the handbook that ensures that new users don't miss anything crucial  and that helps them to become familiar with the structure of the handbook so that they will find it easier to use and are able to find the information they need when they need it as they go on to work on real-world projects.

---

_**Note:** this article is already largely written. It was written for another purpose, but is now not going to be used so it can be re-purposed as a post. So turnaround for this is likely to be very quick._

**Comment by bph** [comment]
issue created. Please continue discussion there

- https://github.com/WordPress/developer-blog-content/issues/110

This discussion is locked.

**Comment by mburridge** [comment]
Post has been [published](https://developer.wordpress.org/news/2023/06/navigating-the-block-editor-handbook/)
